### PR TITLE
fireside:1.0.0

### PR DESCRIPTION
**Pull request by edgarogh**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This template looks a bit friendlier than a pure black-and-white letter, when the context allows it. I've used it successfully as a cover letter on multiple occasions previously. It has a modern look with a warm white background.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/fireside/1.0.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,30 @@
+#import "@preview/fireside:1.0.0": fireside
+
+#show: fireside.with(
+  title: [Anakin \ Skywalker],
+  from_details: [
```
Comment (laurmaedje) [inline packages/preview/fireside/1.0.0/template/main.typ:5]: I recommend naming things in kebab-case, i.e. `from-details`, as it's the idiomatic casing convention for Typst.

Comment (edgarogh) [reply packages/preview/fireside/1.0.0/template/main.typ:5]: Good catch!

File: packages/preview/fireside/1.0.0/lib.typ [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        inset: (top: 1em),
+        [
+          #set text(size: 10.2pt, fill: rgb("4d4d4d"))
+          #from-details
+        ]
+      )),
+    )
+    v(title-size)
+    text(size: 9.2pt, to-details)
+    v(title-size)
```
Comment (elegaanz) [inline packages/preview/fireside/1.0.0/lib.typ:39]: These two `title-size` should become `_title-size` too.
```suggestion
    v(_title-size)
    text(size: 9.2pt, to-details)
    v(_title-size)
```

Comment (edgarogh) [reply packages/preview/fireside/1.0.0/lib.typ:39]: Oops

**Comment by elegaanz** [comment]
Thanks :)

**Pull request merged**